airbyte-integrations/bases/source-acceptance-test/unit_tests/test_backward_compatibility.py

```diff
@@ -915,6 +915,7 @@ def test_spec_backward_compatibility(previous_connector_spec, actual_connector_s
 ]
 
 
+@pytest.mark.slow
```

Comment:
what's the value of this mark?

Reply:
The decorated test is slow to run,  SAT developers can decide to bypass it while iterating on their change by running `python -m pytest unit_tests -m "not slow"`

Reply:
is that a known pythonic standard? might be helpful to include in the docs

Reply:
I thought it was a standard as it's the first example in pytest doc about marks:
https://docs.pytest.org/en/7.1.x/how-to/mark.html#registering-marks

Reply:
I am by no means the yardstick of pythonic knowledge, so take this with a grain of salt, but if there's a chance users may not know about it, it's helpful to add to the docs

Reply:
👍 Sure, if you ask, it must be documented.